posts/zzz_DO_NOT_EDIT_1.2_admiral.../1.2_admiral_release.qmd

```diff
@@ -0,0 +1,277 @@
+---
+title: "{admiral} 1.2 is here!"
+author:
+  - name: Ben Straub
+  - name: Edoardo Mancini
+description: "What to expect from the 1.2 release of {admiral}? New functions, crisper documentation, upcoming plans and more!"
+# Note that the date below will be auto-updated when the post is merged.
+date: "2025-01-20"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [ADaM, Community, Technical]
+# Feel free to change the image
+image: "admiral.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_1.2_admiral..."
+
+# Please use the {link} package to automatically link packages and functions.
+# To link to a package simply wrap it in curly braces: {dplyr}.
+# Link a function from a package like: dplyr::filter().
+# For more information check out {link}.
+library(link)
+link::auto(keep_pkg_prefix = FALSE)
+```
+
+<!--------------- post begins here ----------------->
+
+# Introduction
+
+As crazy as it may sound, the {admiral} package is nearing its third CRAN birthday: our first release dates all the way back to __17th February 2022__! Time really does fly when you're programming ADaMs in R. The codebase has grown quite a lot! For some context, if you'd only allowed yourself to write twenty lines of code a day, starting from the 17th February 2022, by now you would have compiled the whole of the {admiral} codebase (excluding documentation and unit tests). And so, what better early birthday present for {admiral} than the new and exciting 1.2 release! 
+
+Read through this blog post to find out more about the exciting contents of the 1.2 release, as well some of our plans for 2025.
+
+# New Features for 1.2
+
+As always, we have listed the full set of 1.2 updates in our [changelog](https://pharmaverse.github.io/admiral/news/index.html), and have picked out only a select few new functions to discuss.
+
+The three new functions available in this release are:
```

Comment:
```suggestion
To commence, for the first time since {admiral} 1.0 we have added new functions! The three new functions available in this release are:
```